vue/components/ui/templates/home-color-part/home-color-part.vue

```diff
@@ -1,9 +1,9 @@
 <template>
     <div class="home-color-part" v-bind:class="classes" v-bind:style="style" v-if="visible">
         <div class="top">
-            <router-link class="logo-container" v-bind:to="homeRoute">
+            <div class="logo-container" v-on:click="onLogoClick">

```

Comment:
Won't this change the behaviour? Won't this lose the ability to redirect to `homeRoute`?
From what I'm seeing you are just adding a click event and not redirecting on click

Reply:
No
This is being used in [Orchestra](https://ripe-orchestra-ui-platforme.vercel.app/), [Sputnik](https://ripe-sputnik-ui-platforme.vercel.app/), [White admin](https://ripe-white-admin-ui-now.platforme.com/) and [Pulse](https://ripe-pulse-sbx.platforme.com/). In all cases, the the `homeRoute` is the same as the `home-par-color` component therefore not being impacted by removing `router-link` dependendy

Reply:
Don't know if you saw this: https://github.com/ripe-tech/ripe-components-vue/pull/586#issuecomment-1112042672
Other than that, looks good then! ;)